docs/api-reference/1_5/documents/_overview.md

```diff
@@ -0,0 +1,92 @@
+# <strong>API OVERVIEW</strong>
+
+Welcome to the Kubernetes API.  You can use our API to access Kubernetes API endpoints to read and write Kubernetes
```

Comment:
"You can use the Kubernetes API to read and write Kubernetes resource objects via a Kubernetes API endpoint."